src/BenchmarksApps/TodosApi/TodosApi.json

```diff
@@ -58,6 +64,16 @@
                 }
               }
             }
+          },
+          "201": {
```

Comment:
Similarily, it looks like annotations from `IEndpointMetaataProvider` implementations weren't applied before.